ui/litellm-dashboard/src/components/add_model/conditional_public_model_name.tsx

```diff
@@ -71,9 +69,38 @@ const ConditionalPublicModelName: React.FC = () => {
 
   if (!showPublicModelName) return null;
 
+  const publicNameTooltipContent = (
+    <>
+      <div className="mb-2 font-normal">
+        The name you specify in your API calls to LiteLLM Proxy
+      </div>
+      <div className="mb-2 font-normal">
+        <strong>Example:</strong> If you choose <code className="bg-gray-700 px-1 py-0.5 rounded text-xs">openai/qwen-plus-latest</code> as the LiteLLM model, and name your public model <code className="bg-gray-700 px-1 py-0.5 rounded text-xs">example-name</code>
+      </div>
+      <div className="mb-2 font-normal">
+        <strong>Usage:</strong> You make an API call to the LiteLLM proxy with <code className="bg-gray-700 px-1 py-0.5 rounded text-xs">model = "example-name"</code>
```

Comment:
see this sentence here, you got it correct - start with public model name then talk about litellm model name